environment.yml

```
@@ -7,7 +7,7 @@ dependencies:
   - scipy>=1.11.4
   - networkx>=3.0
   - pillow>=10.1
-  - imageio>=2.33,!=2.35.0
+  - imageio>=2.33.1,!=2.35.0
```

Revert now we've got a better candidate?

```suggestion
  - imageio>=2.33,!=2.35.0
```